tasks/src/image-classification/data.ts

```diff
@@ -0,0 +1,71 @@
+import type { TaskData } from "../Types";
+
+import { PipelineType } from "../../../widgets/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Benchmark dataset used for image classification, with images that belong to 100 classes",
+			id:          "cifar100",
+		},
+        {
+			// TODO write proper description
+			description: "Dataset consisting of images of garments",
+			id:          "fashion-mnist",
+		}
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "image-classification-input.jpg",
+				type:     "img",
+			},
+		],
+		outputs: [
+			{
+				filename: "A JSON response including class labels",
+				type:     "img",
+			},
+		],
+	},
+	id:        "image-classification",
+	label:     PipelineType["image-classification"],
+	libraries: TASKS_MODEL_LIBRARIES["image-classification"],
+	metrics:   [
+		{
+			description: "",
+			id:          "Accuracy",
```

Comment:
nit: this and the ones below need to be consistent with https://huggingface.co/metrics I think. So it should be accuracy.